src/components/Stage.tsx

```diff
@@ -26,13 +35,25 @@ const getHabitatUnlockText = (state: GameState, uids: (HabitatUID | DisasterUID
   return `Congratulations!\nYou earned the ${habitatText} ${habitatCount > 1 ? "habitats" : "habitat"}`;
 };
 
+const getCardBoughtText = (state: GameState, uid: AnimalUID | PlantUID | ElementUID | DisasterUID | undefined) => {
+  if (!uid) {
+    return "";
+  }
+
+  const player = find(state.players, { uid: state.turn.player })!;
+  const cardName = find(player.hand, { uid })?.name;
+
+  return `Congratulations!\nYou bought a ${cardName}`;
+};
+
```

Comment:
IMO we should use this thing to get something from state:

https://stately.ai/docs/xstate-react#useselectoractorref-selector-compare-getsnapshot